### PR TITLE
CLI testing

### PR DESCRIPTION
**Pull request by SanojPunchihewa**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Comment by a5anka** [comment]
Shall we configure Travis & Jenkins to run these tests?

**Comment by a5anka** [comment]
@SanojPunchihewa Can you rebase with master? Looks like there are extra changes.

File: .travis.yml [hunk 1]
```diff
@@ -12,15 +12,29 @@
...
-  - docker
-  
-install: true
+      env:
+        - GO111MODULE=on
 
-script: docker run -v "$HOME/.m2":/root/.m2 --rm -v "$(pwd)":/usr/src/mymaven -w /usr/src/mymaven a5anka/wso2-ei-maven mvn clean install -Dsurefire.useSystemClassLoader=false -fae -B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+      before_install:
+        - cd cmd/
+        - go mod download
```
Comment (a5anka) [inline .travis.yml]: If we cache the module download, will it speedup the build?

Comment (SanojPunchihewa) [reply .travis.yml]: Yes, I have updated the code in dda2e3a

**Pull request merged**